docs/guides/admin/docs/releasenotes.md

```diff
@@ -2,6 +2,12 @@
 
 ## Opencast 15.6
 
+**Attention:**
+The fix for the handling of captions by the select-tracks operation
+([5829](https://github.com/opencast/opencast/pull/5829)) requires the removal of the operation directly afterwards from
+the `partial-publish` community workflow, otherwise captions will be duplicated during publication (see
+[6046](https://github.com/opencast/opencast/pull/6046) which is _not_ included in this release).
```

Comment:
I find this wording pretty confusing. Maybe something like this is clearer:
```suggestion
the `partial-publish` workflow, otherwise captions will be duplicated during publication.
The workflow has beein fixed in 15.7.
For details, see [#6046](https://github.com/opencast/opencast/pull/6046).
```

Reply:
> The workflow has been fixed in 15.7.

That's not true though. It will hopefully be fixed in 15.8, but that depends on whether this PR gets merged in time. But I guess one could update the release notes afterwards similarly to how I did it for 16.